src/main/java/com/exercise/tasksmanagementsystem/controller/TaskController.java

```diff
@@ -0,0 +1,31 @@
+package com.exercise.tasksmanagementsystem.controller;
+
+import com.exercise.tasksmanagementsystem.entity.Task;
+import com.exercise.tasksmanagementsystem.repository.TaskRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class TaskController {
+
+    @Autowired
+    private TaskRepository taskRepository;
+
+    @PostMapping("/save")
+    public ResponseEntity<Task> saveTask(@RequestBody Task task){
+        Task savedTask = taskRepository.save(task);
+        return new ResponseEntity<>(savedTask, HttpStatus.OK);
+    }
+
+    @GetMapping("/tasks")
+    public ResponseEntity<List<Task>> listAllTasks(){
+        List<Task> tasks = taskRepository.findAll();
```

Comment:
for (Task task: tasks) {
TaskDto dto = new TaskDto();
dto.setName(task.getName());
}